spec/collections_publisher/publishing_parent_and_child_topic_spec.rb

```diff
@@ -0,0 +1,58 @@
+feature "Publishing a parent and child topic on Collections Publisher", collections_publisher: true do
+  include CollectionsPublisherHelpers
+
+  let(:parent_title) { title_with_timestamp }
+  let(:parent_slug) { slug_with_timestamp }
+  let(:link) { "/topic/" + parent_slug }
+
+  let(:child_slug) { slug_with_timestamp }
+  let(:child_title) { title_with_timestamp }
+
+  scenario "Publishing a parent and child topic" do
+    given_i_have_a_published_topic
+    when_i_add_a_child_topic
+    and_i_publish_it
+    then_i_can_view_it_on_gov_uk
```

Comment:
Since this test is "Publishing a parent and child topic" do we want to check we can view the parent? we could do this by navigating to it from the child?